src/Changelog.ts

```diff
@@ -41,8 +42,10 @@ export default class Changelog {
     );
   }
 
-  sortReleases() {
-    this.releases.sort((a, b) => a.compare(b));
+  sortReleases(force?: boolean) {
```

Comment:
It's a bit confusing to have a function called `sortReleases()` that does nothing.
`autoSortReleases` should be independent to this function. Due this function is only invoked in two places I think it's better to use an `if` in these places.

For example:

```js
if (this.autoSortReleases) {
  this.sortReleases();
}
```

Reply:
It first i thought about doing it this way but then i went with this implementation to see what you will think.
The way you proposed probably would be best.
Also should i add command line option to bin.ts ?

Reply:
Yup, something like `--no-sort` or similar.

Reply:
```
deno 2.1.10 (stable, release, x86_64-unknown-linux-gnu)
v8 13.0.245.12-rusty
typescript 5.6.2
```
This is version of deno which i used to run `deno fmt`

I also removed section with available options from `README.md` so we don't duplicate descriptions in `bin.ts` but whether thats good it's up to you.
Probably `2.6.0` date should be changed